catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -221,6 +221,8 @@
         </property>
     </bean>
 
+    <service ref="endpointUtil" interface="org.codice.ddf.catalog.ui.util.EndpointUtil" />
```

Comment:
Remove this, por favor

Reply:
I need it.

Reply:
I was confused by this initially then realized it was necessary for the `@Reference` in your command. Consider adding a comment to that effect.